src/pke/include/constants.h

```diff
@@ -37,6 +37,10 @@
 // TODO: Review whether we need to include hal.h.
```

Comment:
Agree with this TODO.  I am envisioning that this file should be as lightweight as possible...